ibis/expr/types/relations.py

```diff
@@ -1797,21 +1837,13 @@ def mutate(
         │ Adelie  │  2007 │       -7.22193 │
         └─────────┴───────┴────────────────┘
         """
-        import ibis.expr.analysis as an
-
-        exprs = [] if exprs is None else util.promote_list(exprs)
-        exprs = itertools.chain(
-            itertools.chain.from_iterable(
-                util.promote_list(_ensure_expr(self, expr)) for expr in exprs
-            ),
-            (
-                e.name(name)
-                for name, expr in mutations.items()
-                for e in util.promote_list(_ensure_expr(self, expr))
-            ),
-        )
-        mutation_exprs = an.get_mutation_exprs(list(exprs), self)
-        return self.select(mutation_exprs)
+        # string and integer inputs are going to be coerced to literals instead
+        # of interpreted as column references like in select
+        node = self.op()
+        values = bind(self, (exprs, mutations), prefer_column=False)
+        values = unwrap_aliases(values)
+        values = {**node.fields, **values}
+        return self.select(**values)
```

Comment:
```suggestion
        return self.select(**node.fields, **values)
```

Just chuck 'em all in there in one call!